packages/cli/package.json

```diff
@@ -41,6 +41,7 @@
   "main": "lib/index.js",
   "typings": "lib/index.d.ts",
   "scripts": {
+    "prepublishOnly": "npm i --package-lock-only && npm shrinkwrap",
```

Comment:
I do not remember the addition of the prepublishOnly script in the various plugin PRs. Maybe I just overlooked it. If not, why here and not in the plugins? This is not an objection. I am just trying to understand.

Reply:
In the case of the CLI, since we only want the `packages/cli` package in the monorepo to have an npm-shrinkwrap before publishing, we added the prepublishonly in order to create one based on the already installed dependencies thanks to the top level npm-shrinkwrap file 😋